src/platform/packages/shared/kbn-es/src/serverless_resources/project_roles/security/roles.yml

```
@@ -580,6 +588,7 @@ soc_manager:
         - feature_siemV3.execute_operations_all
         - feature_siemV3.scan_operations_all
         - feature_siemV3.workflow_insights_all
+        - feature_siemV3.soc_management_all
```

just an fyi: i have a PR aiming to update `siemV3` to `siemV4` in kibana repo. i guess github will indicate the conflict for the second PR after one is merged, but to be on the safe side, just wanted to share so we don't forget about it:
- #238635

---

Hm... should I be adding both or just one here?
```suggestion
        - feature_siemV3.soc_management_all
        - feature_siemV4.soc_management_all
```